pkgs/development/compilers/gcc/builder.sh

```diff
@@ -251,6 +253,16 @@ postInstall() {
         done
     fi
 
+    if type "install_name_tool"; then
```

Comment:
Have not really looked at it, but I still get a cycle when removing the `install_name_tool -id` commands.